src/StringHelper.php

```diff
@@ -619,6 +619,107 @@ public static function rtrim(string|array $string, string $pattern = self::DEFAU
         return preg_replace("#[$pattern]+$#uD", '', $string);
     }
 
+    /**
+     * Returns the portion of the string that lies between the first occurrence of the start string
+     * and the last occurrence of the end string after that.
+     *
+     * If the end string is not provided, it defaults to the value of the start string.
+     *
+     * @param string $string The input string.
+     * @param string $start The string marking the start of the portion to extract.
+     * @param string|null $end The string marking the end of the portion to extract.
```

Comment:
```suggestion
     * @param string|null $end The string marking the end of the portion to extract.
     * If not specified, it will default to the value of `$start`.
```

Please move doc about `If the end string is not provided, it defaults to the value of the start string.`
after `@param string|null $end ...`